diff -urN dist.orig/sphinxcontrib/inlinesyntaxhighlight.py dist/sphinxcontrib/inlinesyntaxhighlight.py
--- dist.orig/sphinxcontrib/inlinesyntaxhighlight.py	2017-09-16 13:27:20.825601390 +0900
+++ dist/sphinxcontrib/inlinesyntaxhighlight.py	2017-09-16 13:31:05.055603526 +0900
@@ -40,7 +40,7 @@
 PRE_DIV_RE = re.compile(r'\s*</pre></div>\s*$')
 
 def html_visit_literal(self, node):
-    env = self.settings.env
+    env = self.builder.env
 
     shall_highlight = False
 
