--- dist.orig/sphinxcontrib/plantuml.py	2020-01-01 10:30:59.000000000 +0900
+++ dist/sphinxcontrib/plantuml.py	2020-03-29 10:31:53.879307941 +0900
@@ -212,7 +212,8 @@
                                                         fileformat),
                                  stdout=f, stdin=subprocess.PIPE,
                                  stderr=subprocess.PIPE,
-                                 cwd=absincdir)
+                                 cwd=absincdir,
+                                 env=_new_env())
         except OSError as err:
             if err.errno != ENOENT:
                 raise
@@ -236,7 +237,8 @@
                              stdin=subprocess.PIPE,
                              stdout=subprocess.PIPE,
                              stderr=subprocess.PIPE,
-                             cwd=absincdir)
+                             cwd=absincdir,
+                             env=_new_env())
     except OSError as err:
         if err.errno != ENOENT:
             raise
@@ -395,13 +397,15 @@
     try:
         try:
             p = subprocess.Popen(args, stdout=subprocess.PIPE,
-                                 stderr=subprocess.PIPE)
+                                 stderr=subprocess.PIPE,
+                                 env=_new_env())
         except OSError as err:
             # workaround for missing shebang of epstopdf script
             if err.errno != getattr(errno, 'ENOEXEC', 0):
                 raise
             p = subprocess.Popen(['bash'] + args, stdout=subprocess.PIPE,
-                                 stderr=subprocess.PIPE)
+                                 stderr=subprocess.PIPE,
+                                 env=_new_env())
     except OSError as err:
         if err.errno != ENOENT:
             raise
@@ -515,6 +519,17 @@
 }
 
 
+def _new_env():
+    env = dict(os.environ)  # make a copy of the environment
+    lp_key = 'LD_LIBRARY_PATH'  # for Linux and *BSD.
+    lp_orig = env.get(lp_key + '_ORIG')  # pyinstaller >= 20160820 has this
+    if lp_orig is not None:
+        env[lp_key] = lp_orig  # restore the original, unmodified value
+    else:
+        env.pop(lp_key, None)  # last resort: remove the env var
+    return env
+
+
 def setup(app):
     app.add_node(plantuml, **_NODE_VISITORS)
     app.add_directive('uml', UmlDirective)
