diff -urN dist.orig/sphinx_rtd_theme/__init__.py dist/sphinx_rtd_theme/__init__.py
--- dist.orig/sphinx_rtd_theme/__init__.py	2023-02-20 21:42:42.078480089 +0900
+++ dist/sphinx_rtd_theme/__init__.py	2023-02-20 21:46:46.426078882 +0900
@@ -6,6 +6,7 @@
 
 from os import path
 from sys import version_info as python_version
+import sys
 
 from sphinx import version_info as sphinx_version
 from sphinx.locale import _
@@ -20,7 +21,10 @@
 
 def get_html_theme_path():
     """Return list of HTML theme paths."""
-    cur_dir = path.abspath(path.dirname(path.dirname(__file__)))
+    if getattr(sys, 'frozen', False):
+        cur_dir = sys._MEIPASS
+    else:
+        cur_dir = path.abspath(path.dirname(path.dirname(__file__)))
     return cur_dir
 
 
