diff -urN dist.orig/sphinx_rtd_theme/__init__.py dist/sphinx_rtd_theme/__init__.py
--- dist.orig/sphinx_rtd_theme/__init__.py	2019-02-13 04:37:55.000000000 +0900
+++ dist/sphinx_rtd_theme/__init__.py	2019-04-28 11:02:47.203040056 +0900
@@ -5,6 +5,7 @@
 """

 from os import path
+import sys

 from sphinx import version_info
 from sphinx.locale import _
@@ -24,7 +25,10 @@

 def get_html_theme_path():
     """Return list of HTML theme paths."""
-    cur_dir = path.abspath(path.dirname(path.dirname(__file__)))
+    if getattr(sys, 'frozen', False):
+        cur_dir = sys._MEIPASS
+    else:
+        cur_dir = path.abspath(path.dirname(path.dirname(__file__)))
     return cur_dir
 
 
