diff -urN dist.orig/sphinxemoji/sphinxemoji.py dist/sphinxemoji/sphinxemoji.py
--- dist.orig/sphinxemoji/sphinxemoji.py	2019-07-11 09:13:36.000000000 +0900
+++ dist/sphinxemoji/sphinxemoji.py	2019-07-26 23:40:15.282033254 +0900
@@ -23,7 +23,7 @@
         config = self.document.settings.env.config
         settings, source = self.document.settings, self.document['source']
         codes = resource_filename(__name__, 'codes.json')
-        replacements = json.load(open(codes))
+        replacements = json.load(open(codes, encoding='utf-8-sig'))
         to_handle = (set(replacements.keys()) -
                      set(self.document.substitution_defs))
 
